corehq/apps/export/static/export/js/export_list.js

```diff
@@ -185,6 +187,31 @@ hqDefine("export/js/export_list", [
             });
         };
 
+        //button hover text
+        function tooltipStyle(color, margin) {
+            $('.tooltip-inner').css({'background-color': color, 'color': 'black', 'margin-right': margin})
+            $('.tooltip.top .tooltip-arrow').css({'border-top-color': color});
+        }
+
+        var popoverText = "";
+        if (self.isOData() || self.isFeed()) {
+            popoverText = "All of the selected feeds will be deleted.";
+        }
+        else {
+            popoverText = "All of the selected exports will be deleted.";
+        }
+
+        $('#bulk-delete-text').tooltip({
+            trigger: 'hover',
+            placement: 'top',
+            title: gettext(popoverText),
```

Comment:
I got rid of the inline css in the javascript but opted to have the style outlined in the html its used in, since I couldn't find which stylesheet (if any) were affecting the bootstrap tooltips nor create any new styles in any of the css files I suspected it might be using and have it apply to the tooltips.